tests/integration/testlib/assertlib.py

```diff
@@ -34,3 +35,10 @@ def assert_state(desired_state_data):
     full_desired_state.normalize()
 
     assert full_desired_state.state == current_state.state
+
+
+def assert_absent(ifname):
```

Comment:
Can you use a list of ifaces as input? `*ifnames` could work nicely here.

Reply:
fixed this